components/eamxx/src/share/atm_process/atmosphere_process_hash.cpp

```diff
@@ -129,9 +129,9 @@ ::print_global_state_hash (const std::string& label, const bool in, const bool o
   if (m_comm.am_i_root())
     for (int i = 0; i < nslot; ++i)
       if (show[i])
-        fprintf(stderr, "exxhash> %4d-%9.5f %1d %16lld (%s)\n",
+        fprintf(stderr, "exxhash> %4d-%9.5f %1d %16lx (%s)\n",
```

Comment:
Nitpicky: I would use `%16llx` as format (notice the double ell). The hash is a 64 bit integer, so we want to be sure we are using a 64 bit format integer in the print statement, to avoid compiler warnings.

Yes, on most 64bit systems, `long` _is_ 64 bits, but technically it is still implementation dependent; so you _could_ find a system where `long` is 32 bits, causing warnings.

Reply:
Agreed.  I tested again with same test and verified identical behaviour.